src/app/Http/Controllers/Query/SwitcherController.php

```diff
@@ -547,7 +556,8 @@ private function getServerInfo($request): BaseJsonResponse
             'auto_geocoding_enabled' => legacy_config('auto_geocoding_enabled'),
             'commit' => config('app.commit'),
             'default_closed_status' => legacy_config('default_closed_status'),
-            'aggregator_mode_enabled' => legacy_config('aggregator_mode_enabled')
+            'aggregator_mode_enabled' => legacy_config('aggregator_mode_enabled'),
+            'server_ip' => $serverIp
```

Comment:
There may be a desire to opt-out of this since there is some risk associated with this.

Reply:
Also if there is an error maybe `$serverIp` should be empty or null, rather than returning the error in the response.  The error should be logged though.

Reply:
This is in support of a hosted geocoding and time zone API that Patrick is working on. The idea is to use this endpoint to gather IPs from the servers in the tally for allowlisting. We do actually want to return the error here for troubleshooting, as we won't actually have access to most of the root servers.